manifests/components/elasticsearch-curator.jsonnet

```diff
@@ -0,0 +1,128 @@
+/*
+ * Bitnami Kubernetes Production Runtime - A collection of services that makes it
+ * easy to run production workloads in Kubernetes.
+ *
+ * Copyright 2018-2019 Bitnami
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+local kube = import '../lib/kube.libsonnet';
+local utils = import '../lib/utils.libsonnet';
+
+local CURATOR_IMAGE = (import 'images.json').curator;
+
+// Implement elasticsearch-curator as a Kubernetes CronJob
+local elasticsearch_curator = {
+  namespace:: 'kubeprod',
+  name:: 'elasticsearch-curator',
+  retention_days:: 60,
+  elasticsearch_host:: 'elasticsearch-logging',
```

Comment:
Wonder if we should leave these two instead as
```
error "elasticsearch_host must be externally provided ..."
```
then set it from the main ES object [*1]

Reply:
Addressed